android/src/main/res/layout/activity_about.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Shouldn't we do this with ConstraintLayout instead?

Reply:
We could, but if it works fine and it's not a complicated layout, I think it is best to keep it as-is now.